### PR TITLE
update sha* alg definitions

### PR DESCRIPTION
**Pull request by omsmith**
Working on updating the test fixtures to codify it, but do believe these should all support both ecdsa and rsa. Should be the "correct" ones to use vs the "RSA-SHA*" aliases.

Ref: https://github.com/brianloveswords/node-jwa/pull/26

### CODE REVIEW

**Comment by calvinmetcalf** [comment]
ok so the key test would be one that 

1. works in current node
2. didn't work previously here
3. works with this change

File: test/index.js [hunk 1]
```diff
@@ -25,23 +29,30 @@ fixtures.valid.rsa.forEach(function (f) {
...
 
-    t.equals(bSig.length, nSig.length, 'correct length')
-    t.equals(bSig.toString('hex'), nSig.toString('hex'), 'equal sigs')
-    t.equals(bSig.toString('hex'), f.signature, 'compare to known')
+      t.equals(bSig.length, nSig.length, 'correct length')
+      t.equals(bSig.toString('hex'), nSig.toString('hex'), 'equal sigs')
 
-    t.ok(nCrypto.createVerify(f.scheme).update(message).verify(pub, nSig), 'node validate node sig')
-    t.ok(nCrypto.createVerify(f.scheme).update(message).verify(pub, bSig), 'node validate browser sig')
+      t.equals(nSig.toString('hex'), f.signature, 'node compare to known')
```
Comment (omsmith) [inline test/index.js]: Tests were failing before I added the id/tag to the other definitions, so I added this as a sanity check. This was consistently "OK", meaning node was doing the expected thing.

File: browser/algorithms.json [hunk 2]
```diff
@@ -50,24 +50,24 @@
     "id": ""
   },
   "sha256": {
-    "sign": "ecdsa",
+    "sign": "ecdsa/rsa",
     "hash": "sha256",
-    "id": ""
+    "id": "3031300d060960864801650304020105000420"
```
Comment (omsmith) [inline browser/algorithms.json]: Added these to get RSA going, which is the only thing it impacts (as I'm sure you know). Although, it is a little weird to do

Comment (calvinmetcalf) [reply browser/algorithms.json]: yeah this file was just set up before I latter learned how weird some of the names were handled in node

File: test/index.js (hunk 1 above)
Comment (calvinmetcalf) [reply test/index.js]: ok this is a good check

**Pull request closed without merging**